uber/models.py

```diff
@@ -2063,8 +2079,18 @@ def is_expired(cls):
         return cls.expiration_date < localized_now()
 
     @hybrid_property
+    def is_free(self):
+        return not self.discount or (
+            self.discount_type == self.PERCENT_DISCOUNT and self.discount > 99)
```

Comment:
This is odd, was there a reason you didn't just say `self.discount >= 100`? Does 99.3 get rounded up in our calculations or something?